it/helpers/ComponentSpecBase.scala

```diff
@@ -269,6 +269,8 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
 
     def getPaymentHistory: WSResponse = get(s"/payment-refund-history")
 
+    def getMessagesCheck: WSResponse = get(s"/test-only/message-check")
```

Comment:
these are redundant as we are not testing testOnly controller anyway

Reply:
Language-specific keys are added in both the message and message.cy files so that some keys might be available in the message and the same won't be there in messacy.cy(blank) and vice versa. I thought of that scenario and designed both routes to test.